src/runtime/hvx_sysmon.cpp

```diff
@@ -0,0 +1,44 @@
+#include "runtime_internal.h"
+#include "HalideRuntimeQurt.h"
+
+extern "C" {
+__attribute__((weak)) void HP_profile(unsigned int marker, unsigned char enable);
+WEAK uint32_t halide_qurt_sysmon_enabled = 0;
```

Comment:
Put these in a namespace (not HP_profile of course) and remove the halide_qurt_ prefix. Convention in other runtime modules is:

    namespace Halide { namespace Runtime { namespace Internal { namespace Qurt

e.g.: https://github.com/halide/Halide/blob/master/src/runtime/mini_qurt.h#L5